lib/src/flutter_local_notifications.dart

```diff
@@ -122,7 +122,8 @@ class FlutterLocalNotificationsPlugin {
       'title': title,
       'body': body,
       'platformSpecifics': serializedPlatformSpecifics,
-      'payload': payload ?? ''
+      'payload': payload ?? '',
+      'ticker' : ticker
```

Comment:
given this is an Android-specific setting, instead of exposing it as a parameter, could you move this setting to the Android-specific _AndroidNotificationDetails_ class? this will making more changes as a result of this

Reply:
also upon moving it to the _AndroidNotificationDetails_ class, could you add some documentation on the new field? could almost be a copy and paste of what's written in the Android docs